docs/notebooks/dispersion/two_fluid_dispersion_solver.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<p>Replace the top cell with a markdown cell containing the notebook title.  Something like <code>Dispersion: Two Fluid Analytical Solution</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/932/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>